src/main/webapp/app/shared/util/firebase/firebase-review-utils.tsx

```diff
@@ -688,7 +705,7 @@ export const buildStringReview = (
 };
 
 export const buildObjectReview = (
-  obj: Record<string, any>,
+  obj: object | Implication | null,
```

Comment:
I got it down to just `obj: object`. I don't remember why I added `Implication` but I removed it too. As far as null goes, there's a check [here](https://github.com/oncokb/oncokb-transcript/blob/rc/src/main/webapp/app/shared/util/firebase/firebase-review-utils.tsx#L436) that looks to see if the type of the object is `object`. Since we're in the world of JavaScript null is an a typeof object so it doesn't guard against null. I added a null check to that if statement too.